addon/route-handlers/shorthands/delete.js

```diff
@@ -13,12 +13,14 @@ export default class DeleteShorthandRouteHandler extends BaseShorthandRouteHandl
   handleStringShorthand(modelName, schema, request) {
     let id = this._getIdForRequest(request);
     let type = camelize(modelName);
+    let schem = schema[type];
```

Comment:
could we rename `schem` to `modelClass` here (and through the rest of the PR)? I think it's a bit more descriptive